video/out/opengl/context.c

```diff
@@ -91,6 +91,14 @@ static const struct mpgl_driver *const backends[] = {
 
 static bool get_desc(struct m_obj_desc *dst, int index)
 {
+    if (index == 0) {
+        *dst = (struct m_obj_desc) {
+            .name = "auto",
+            .description = "probe for correct direver"
```

Comment:
heh, "direver"

As for the distinction between driver, device, backend, window... it was all unclear in the code before I started working on this. It's referred to as both a driver and a backend in the code. In the docs it's referred to as a window and a backend. I think I'll use backend here.